### ISSUE TITLE
[Router] Does router supports parameters?

### ISSUE THREAD

**Issue by xperiandri** [fsbolero/Bolero#309]
Like
``` F#
type Page =
    | [<EndPoint "/">] Home
    | [<EndPoint "/counter?value={Value}">] Counter of Value: int
    | [<EndPoint "/data/{item}">] Data of Guid
```

**Comment by Tarmil** [comment]
Currently, no. It was initially prevented by limitations in Blazor's `NavigationManager`. But I see that `GetUriWithQueryParameter` has since been added, which suggests that this might now be possible.

**Comment by Tarmil** [comment]
This has now been released in v0.23.

**Comment by xperiandri** [comment]
Super cool!
